zb-db/src/main/java/io/zeebe/db/impl/rocksdb/RocksDbConfiguration.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Zeebe Community License 1.0. You may not use this file
+ * except in compliance with the Zeebe Community License 1.0.
+ */
+package io.zeebe.db.impl.rocksdb;
+
+import java.util.Properties;
+
+public final class RocksDbConfiguration {
+  public static final int DEFAULT_UNLIMITED_MAX_OPEN_FILES = -1;
+  public static final int DEFAULT_MAX_OPEN_FILES = 2048;
```

Comment:
I think we can have only one setting, that's fine. And just use it below, adapting the comment a little.